build/index.js

```diff
@@ -44,11 +44,11 @@
   });
 
   app.use(function*(next) {
-    var err;
+    var err, error;
```

Comment:
I didn't change this file myself, it got changed to this when I ran the `gulp coffee` task I think.